code/datums/materials/basemats.dm

```diff
@@ -109,17 +113,23 @@
 	categories = list(MAT_CATEGORY_ORE = TRUE, MAT_CATEGORY_RIGID = TRUE)
 	sheet_type = /obj/item/stack/sheet/mineral/bananium
 	coin_type = /obj/item/coin/bananium
+	armor_modifiers = list("melee" = 0, "bullet" = 0, "laser" = 0, "energy" = 0, "bomb" = 100, "bio" = 0, "rad" = 0, "fire" = 10, "acid" = 0) //Clowns cant be blown away.
 
 /datum/material/bananium/on_applied(atom/source, amount, material_flags)
 	. = ..()
 	source.AddComponent(/datum/component/squeak, list('sound/items/bikehorn.ogg'=1), 50)
 	source.AddComponent(/datum/component/slippery, min(amount / 10, 80))
 
+/datum/material/bananium/on_applied_obj(var/obj/o, amount, material_flags)
+	o.AddComponent(/datum/component/wearertargeting/wearer_crosser, list(SLOT_HEAD, SLOT_SHOES, SLOT_BACK, SLOT_W_UNIFORM, SLOT_WEAR_SUIT))
```

Comment:
This is scuffed. wearertargeting only allows items but this tries to add it to any obj.